internal/endpoint/machineService.go

```diff
@@ -43,12 +52,11 @@ func (h *endpointHandler) NewMachineService() *restful.WebService {
 
 	ws.Route(ws.POST("/report/{id}").
 		To(h.Report).
-		Doc("report machine by ID").
+		Doc("report machine").
 		Param(ws.PathParameter("id", "identifier of the machine").DataType("string")).
 		Metadata(restfulspec.KeyOpenAPITags, tags).
 		Reads(domain.Report{}).
-		Writes(domain.BootResponse{}).
-		Returns(http.StatusOK, "OK", models.V1MachineResponse{}).
+		Returns(http.StatusOK, "OK", nil).
```

Comment:
Why not returning the MachineResponse to metal-hammer ? This was quite useful to see in the hammer logs what metal-api calculated from the report call, right ?

Reply:
The current implementation always respond nil, thus I made that fact clear through this change. Same with the changes above.

Reply:
OK